BLaurenB/techtime
# Fix cart on nav

#### What's this PR do?
This fixes the nav bar by combing the link to the cart page with the total count. 

#### Where should the reviewer start- what are the pertinent changes and the sequence?
Review the application.html.erb to review the changes to the nav bar.

#### How should this be manually tested?
run test suite and ran on local host to verify the change renders in the browser.

#### Any background context you want to provide?
#### What are the relevant tickets?
closes #109 
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28613896/32534083-2f752130-c411-11e7-99d1-b6a2f7dabf4a.png)

#### Questions:
